pretyping/pretyping.ml

```diff
@@ -518,17 +518,24 @@ let pretype_ref ?loc evdref env ref us =
     let ty = unsafe_type_of env.ExtraEnv.env evd c in
       make_judge c ty
 
-let judge_of_Type ?loc evd s =
-  let evd, s = interp_universe ?loc evd s in
+let judge_of_Type ?loc env evd s =
```

Comment:
Is this directly related to the rest of the changes?

Reply:
It could be an independent PR. I just spotted that it was potentially breaking the no-algebraic  in term position invariant.

Reply:
This closes recent bug reports I believe, where Type@{i+1} is put in the wrong place.